pkgs/applications/audio/parlatype/default.nix

```
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, pkgconfig, meson, gnome3, at-spi2-core, dbus, gst_all_1, sphinxbase, pocketsphinx, ninja, gettext, appstream-glib, python3, glib, gobject-introspection, gsettings-desktop-schemas, itstool, wrapGAppsHook, makeWrapper, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  name = "parlatype-${version}";
+  version = "v1.6-beta";
+
+  src = fetchFromGitHub {
+    owner  = "gkarsay";
+    repo   = "parlatype";
+    rev    = "98fffe19b7a0b9bdc893309321f435338a58423f";
```

```suggestion
    rev    = "v${version}";
```
It is better to use a  tag instead of a git commit, because it allows for automated updates.